tests/test_matrix.py

```diff
@@ -174,7 +174,7 @@ def test_interpolation(
         self, node_type: Any, values: Any, restore_resolvers: Any
     ) -> None:
         resolver_output = 9999
-        OmegaConf.register_resolver("func", lambda: resolver_output)
+        OmegaConf.register_new_resolver("func", lambda: resolver_output)
```

Comment:
Since we support both old and new styles in OmegaConf 2.1, test both.
You can parameterize on the resolver API version, doubling the number of tests here.

Reply:
Done in c886ad4